azurerm/internal/services/network/web_application_firewall_policy_data_source.go

```diff
@@ -0,0 +1,62 @@
+package network
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/azure"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/tags"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/timeouts"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataArmWebApplicationFirewallPolicy() *schema.Resource {
```

Comment:
🤦 sorry just realised I forgot to check for tests in this one - would you mind adding an acceptance tests covering this data source? [here's an example](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/internal/services/resource/tests/data_source_resource_group_test.go#L12-L30)